src/app/components/turnos/gestor-agendas/gestor-agendas.component.ts

```diff
@@ -599,7 +642,67 @@ export class GestorAgendasComponent implements OnInit, OnDestroy {
         this.showGestorAgendas = false;
     }
 
-    actualizarEstadoEmit(estado) {
+    actualizarEstado(estado) {
+
+        switch (estado) {
+            case 'publicada':
+                this.plex.confirm('¿Publicar Agenda?').then((confirmado) => {
+                    if (!confirmado) {
+                        return false;
+                    } else {
+                        this.confirmarEstado(estado);
+                    }
+                });
+                break;
+            case 'suspendida':
+                this.actualizarGestor(estado);
+                break;
+            case 'borrada':
+                this.plex.confirm('¿Borrar Agenda?').then((confirmado) => {
+                    if (!confirmado) {
+                        return false;
+                    } else {
+                        this.confirmarEstado(estado);
+                    }
+                });
+                break;
+            default:
+                this.confirmarEstado(estado);
+                break;
+
+        }
+    }
+
+    confirmarEstado(estado) {
```

Comment:
Me llama la atención este código... por lo visto esta replicado de otras implementaciones. Si bien aún no probé esta funcionalidad en vivo, me hace ruido el patch secuencial y el manejo de mensajes de resultados de la operaciónes. Entiendo que muestra el toast de operación existosa una unica vez para el primer patch con status 200, y luego sigue patcheando las demas. Y eventualmente muestra un alerta si falla alguno de los siguienetes patchs y actualiza el gestor... pero el mensaje es genérico y no indica cual agenda. Se me hace bastante confuso... Tal vez podriamos considerar un PATCH de varios recursos en un request. Que opinan @negro89 @martinebucarey ? Dejo esto por aqui .... https://stackoverflow.com/questions/32098423/rest-updating-multiple-resources-with-one-request-is-it-standard-or-to-be-avo